cvat/apps/dataset_manager/bindings.py

```diff
@@ -797,20 +799,25 @@ def categories(self) -> dict:
         raise NotImplementedError()
 
     @staticmethod
-    def _load_categories(labels: list):
+    def _load_categories(meta: dict, dimension):
         categories: Dict[datumaro.AnnotationType, datumaro.Categories] = {}
 
         label_categories = datumaro.LabelCategories(attributes=['occluded'])
 
-        for _, label in labels:
+        user_info = {}
+        if dimension == DimensionType.DIM_3D:
+            user_info = {"name": meta['owner']['username'],
```

Comment:
Can we make it another function? There is no reason to include it here.

Reply:
Fixed